### DISCUSSION TITLE
Integration with Print Job History plugin

### DISCUSSION THREAD

**Discussion by quloquloqulo** [OctoFarm/OctoFarm#320] [Feature Request]
https://plugins.octoprint.org/plugins/PrintJobHistory/

Ability to keep the plugin and octofarm history synchronized or the ability to import from the plugin would be nice.
It would allow you to recover the prints prior to octofarm or those that are not registered for various problems (network problems, unexpected raspberry restart, etc ..)
the plugin should have its own API

**Comment by NotExpectedYet** [comment]
> https://plugins.octoprint.org/plugins/PrintJobHistory/
> 
> Ability to keep the plugin and octofarm history synchronized or the ability to import from the plugin would be nice.
> It would allow you to recover the prints prior to octofarm or those that are not registered for various problems (network problems, unexpected raspberry restart, etc ..)
> the plugin should have its own API

I'd be up for grabbing the history from PrintJobHistory as a feature to back fill OctoFarms own history but I wouldn't want to go any further than that. Supporting plugins creates a lot of technical debt for OctoFarm as I'm finding with the few I do support, the worst been Filament Manager plugin so I'm hesitant to increase that doing the check and sync for missed print jobs for example.
